### PR TITLE
add IntBetween()

### PR DESCRIPTION
**Pull request by jakobii**
Check out the `numbers_test.go` might help you with idea's for your rand test. 

also, in the `seed.go` file the `init()` func will be run when the package is imported by another, package. So we don't need to set the seed at every function call. Calling `rand.Seed()` in every function call is actually not a good practice, as it could make our rand values more predicable.

### CODE REVIEW

File: numbers_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package rdg
+
+import "testing"
+
+func TestIntBetween(t *testing.T) {
+	t.Parallel()
```
Comment (jakobii) [inline numbers_test.go:6]: these child tests are run in parallel, so they are extra fast! also having them named like this is very helpful for testing edge cases. 
```go
t.Run("describe edge case here", func(t *testing.T){  /* test logic here */ })
```
